pinterest-for-woocommerce.php

```diff
@@ -22,11 +22,11 @@
  * Domain Path:       /i18n/languages
  *
  * Requires at least: 5.6
- * Tested up to: 5.9
+ * Tested up to: 6.2
```

Comment:
@tomalec These bumps can be done during the release process as part of Woorelease rather than by manually changing the versions in a PR.

Reply:
Should I revert it, or could we leave it as is?

Reply:
@JPry but we'd still have to include changelog entry via PR right?

Reply:
@jconroy 

> but we'd still have to include changelog entry via PR right?

Using the Woorelease commands to do the update will also add to the changelog.

@tomalec 

> Should I revert it, or could we leave it as is?

We can leave it since it's already there, but for the future it is easier to bump version headers with Woorelease.